index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+    <title>Nurbol Sultanov</title>
+</head>
+
+<body>
+<!-- ─── Header ─── -->
+    <header class="header">    
+    <nav class="navbar">
+        <div class="logo">
+            <a href="#">My Logo</a>
+        </div>
+        <div class="hamburger">
+            <div class="bar top"></div>
+            <div class="bar middle"></div>
+            <div class="bar bottom"></div>
+          </div>
+            </button>
+              </nav>
+    </header>
+<!-- ─── Headline ─── -->
+        <section class="headline">
```

Comment:
- [Optional] Dear @everinurmind, Good job making the changes and your design is looking great, I suggest the spacing between the header and headline is a bit larger than the Figma design. Addressing this will make your design even better. 
![Screenshot 2023-01-13 223414](https://user-images.githubusercontent.com/69077061/212404275-5dda18c7-9c26-4a0b-8ce5-60b850a1b37e.png)